### PR TITLE
metrics(ticdc): add resolved ts and add changefeed to dataflow

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve? 

* fix etcd worker txn size heatmap
* add owner resolved ts metrics
* add changefeed to dataflow
* add PD uptime
* add PD leader history
* add etcd metrics, eg, mvcc db size, etc.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Import to grafana.

Related changes

 - Need to cherry-pick to the release branch

### Release note 

```release-note
None
```

### CODE REVIEW

File: metrics/grafana/ticdc.json [hunk 1]
```diff
@@ -849,46 +854,30 @@
...
           "renderer": "flot",
           "seriesOverrides": [],
           "spaceLength": 10,
           "stack": false,
-          "steppedLine": false,
+          "steppedLine": true,
           "targets": [
             {
-              "expr": "histogram_quantile(0.999, sum(rate(ticdc_server_etcd_health_check_duration_bucket{tidb_cluster=\"$tidb_cluster\", capture=~\"$capture\"}[1m])) by (le,instance))",
+              "expr": "pd_tso_role{tidb_cluster=\"$tidb_cluster\", dc=\"global\"} > BOOL 0.5",
```
Comment (amyangfei) [inline metrics/grafana/ticdc.json]: Please describe how many changes have been applied to dashboard, since the diff is large enough.

Comment (overvenus) [reply metrics/grafana/ticdc.json]: Updated, thanks for reminding!

**Review by asddongmen — APPROVED** [review]

**Review by amyangfei — APPROVED** [review]

**Pull request merged**